Source/CombatExtended/CombatExtended/Comps/CompFireModes.cs

```diff
@@ -251,6 +301,39 @@ public IEnumerable<Command> GenerateGizmos()
                 LessonAutoActivator.TeachOpportunity(CE_ConceptDefOf.CE_AimModes, parent, OpportunityType.GoodToKnow);
             }
             yield return toggleAimModeGizmo;
+
```

Comment:
Do we want the gizmo hidden if set to suppress?  Or visible but disabled?  I can see a case for either one, so am not picky, but it's worth considering.

Reply:
Likewise for lack of `Handling`, that one might confuse people, especially if the gizmo vanishes mid fight due to an injury.